internal/controller/constants.go

```diff
@@ -72,23 +92,46 @@ const (
 	OLSAppServerServiceName = "lightspeed-app-server"
 	// OLSCertsSecretName is the name of the TLS secret for OLS.
 	OLSCertsSecretName = "lightspeed-tls" // #nosec G101
-	// RedisServicePort is the port number of the OLS redis server service
-	RedisServicePort = 6379
-	// RedisMaxMemory is the max memory of the OLS redis cache
-	RedisMaxMemory = "1024mb"
-	// RedisMaxMemoryPolicy is the max memory policy of the OLS redis cache
-	RedisMaxMemoryPolicy = "allkeys-lru"
+	// PostgresServicePort is the port number of the OLS postgres server service
+	PostgresServicePort = 5432
+	// PostgresSharedBuffers is the share buffers value for postgres cache
+	PostgresSharedBuffers = "256MB"
+	// PostgresMaxConnections is the max connections values for postgres cache
+	PostgresMaxConnections = 2000
+	// PostgresDefaultSSLMode is the default ssl mode for postgres
+	PostgresDefaultSSLMode = "require"
+	// PostgresBootStrapScriptContent is the postgres's bootstrap script content
+	PostgresBootStrapScriptContent = `
+	#!/bin/bash
+	
+	echo "attempting to create pg_trgm extension if it does not exist"
+	
+	_psql () { psql --set ON_ERROR_STOP=1 "$@" ; }
+	
+	echo "CREATE EXTENSION IF NOT EXISTS pg_trgm;" | _psql -d $POSTGRESQL_DATABASE
```

Comment:
i'd expect this to be part of our postgres image, is there a reason you did it this way?

Reply:
Yes. Moving this to postgres image sounds good to me.